packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -411,6 +411,21 @@ const LoggedInPlansFeatureActionButton = ( {
 					</div>
 				) }
 			</Plans2023Tooltip>
+		) : (
```

Comment:
**Note:**

We probably don't need to render a duplicate set of dummy CTA buttons. Fix this before merging.